<launch>
  <!-- Launch the grid map tutorial demo node -->
  <node pkg="grid_map_yh" type="pathPlanning" name="pathPlanning" output="screen" >
  
	<param name="size0" value="10"/>
    <param name="resolution" value ="0.2"/>
    <param name="enableCross" value="false"/>

　　　　<param name="robotFrame" value="velodyne"/>
    <param name="targetMsgName" value="target"/>


  </node>

</launch>
